braidpool_spec.md

```diff
@@ -70,18 +82,50 @@ While we could commit to this data in a more space-efficient manner (e.g. via a
 pubkey tweak), the coinbase is also the location of the `extranonce` 8-byte
 field used by some mining equipment.
 
+[kp]
+
+I don't know why you are talking about the consensus mechanism in the above
+paragraph. We are jumping into these details too quickly. Usually you would:
+
+
+1. Identify all components
+2. Provide an over view of all components
+3. Show how the various components together, i.e. the interfaces between them
+4. Finally you dig into each component.
+
+We are doing 4. here straight away. People will be very confused when reading
+this.
+
+[/kp]
+
+
 The `Coinbase Transaction` is a standard transaction having no inputs, and
 must have the following outputs:
 
     OutPoint(Value:0, scriptPubKey OP_RETURN "BP"+<Braidpool Commitment>+<extranonce>)
     OutPoint(Value:<block reward>, scriptPubKey <P2TR pool_pubkey>)
+	
+[kp]
+
+Why P2TR? Are we just being fancy, or do we really need it? If so, let's
+motivate why and explain it clearly.
+
+[/kp]
 
 The `<block reward>` is the sum of all fees and block reward for this halving
 epoch, and `pool_pubkey` is an address controlled collaboratively by the pool in
 such a way that the [braid consensus mechanism](#braid-consensus-mechanism) can
 only spend it in such a way as to pay all hashers in the manner described by its
 share accounting.
 
+[kp]
+
+How is the block reward calculated? You seem to say it is for all halving. Is it
+simply the sum of all braidpool found blocks in the halving period? We should
+write out a simple formula to capture it precisely.
+
+[/kp]
+
```

Comment:
Here I am describing the sum of all block rewards and fees.

Elsewhere we should describe how shares are paid out. (proportional)

Probably more clarity would be good...